.circleci/config.yml

```diff
@@ -91,8 +91,8 @@ jobs:
           name: Create kubeadm cluster and install machine-controller onto it
           command: ./test/tools/integration/provision_master.sh
       - run:
-          name: Test create and destroy ubuntu machine
-          command: ./test/tools/integration/create-and-destroy-machine.sh
+          name:  Run E2E test
+          command: ./test/tools/integration/run_e2e_tests.sh
       - run:
```

Comment:
The following step needs to be removed ("Test create and destroy centos machine") and moved into the tool

Reply:
`create-and-destroy-machine.sh` exercises `HETZNER` provider, whereas `run_e2e_tests.sh` exercises `DO` and `AWS`

keeping `create-and-destroy-machine.sh` is temporary. I'll create an issue for that. Just wanted to keep this pull small.